homeassistant/components/zwave/__init__.py

```diff
@@ -916,3 +918,16 @@ def refresh_from_network(self):
             return
         for value_id in dependent_ids + [self._value.value_id]:
             self._value.node.refresh_value(value_id)
+
+    @callback
+    def _schedule_update(self):
+        """Schedule delayed update."""
+        @callback
+        def do_update():
+            """Really update."""
+            self.hass.async_add_job(self.async_update_ha_state)
+            self._scheduled_update = False
+
+        if not self._scheduled_update:
```

Comment:
Move if-statement to first line in function so Python doesn't have to create a function for nothing.